### PR TITLE
feat: Specific feedback chat labels

### PR DESCRIPTION
**Pull request by andrewkong9**
Enabled different labels for the different chat tabs available. It is now clear which chat tab is getting feedback (e.g. Translate, Summarize, etc.).

### CODE REVIEW

**Review by haitmo — CHANGES_REQUESTED** [review]
Good job Andrew. Here are a few things that I think should be changed

**Comment by haitmo** [comment]
Thanks Andrew!

Can you also:
- Add a quick test or supplement the existing one to take into account this new label?
- Merge the latest changes from main into this branch

File: django/otto/forms.py [hunk 1]
```diff
@@ -65,7 +64,19 @@ def initialize_chat_feedback(self, message_id):
...
         )
-        self.fields["app"].choices = [("chat", "Chat")]
+
+        chat_mode = Message.objects.get(id=message_id).mode
+        if chat_mode == "translate":
+            self.fields["app"].choices = [("translate", _("Translate"))]
+        elif chat_mode == "summarize":
+            self.fields["app"].choices = [("summarize", _("Summarize"))]
+        elif chat_mode == "document_qa":
+            self.fields["app"].choices = [("document qa", _("Document QA"))]
```
Comment (juleskuehn) [inline django/otto/forms.py:74]: this mode doesn't actually exist anymore

File: .vscode/settings.json [hunk 2]
```diff
@@ -68,5 +68,10 @@
   ],
   // Linux
   "terminal.integrated.defaultProfile.linux": "zsh",
-  "python.REPL.enableREPLSmartSend": false
+  "python.REPL.enableREPLSmartSend": false,
+  "python.testing.pytestArgs": [
+    "django"
+  ],
+  "python.testing.unittestEnabled": false,
+  "python.testing.pytestEnabled": true
```
Comment (juleskuehn) [inline .vscode/settings.json:76]: Not sure why these changes are needed, but I assume they won't cause issues for others. If so, we can remove them in another PR.

**Review by juleskuehn — APPROVED** [review]
Looks good! Feel free to merge.

**Review by juleskuehn — APPROVED** [review]

**Pull request merged**